app/models/project.rb

```diff
@@ -49,6 +49,7 @@ class Project < ApplicationRecord
            source: :user
 
   has_one :code_climate_project_metric, dependent: :destroy
+  has_one :jira_project, dependent: :destroy
```

Comment:
is it possible that there are N JIRAs for 1 project? for example a big project with multiple pods where each pod uses a different JIRA board?

Reply:
for now, we are going to stick with a 1:1 relationship, we talked about this with @santiagovidal yesterday